cura/Settings/MachineManager.py

```diff
@@ -1315,6 +1262,9 @@ def _updateQualityWithMaterial(self, *args: Any) -> None:
         # The current quality type is not available so we use the preferred quality type if it's available,
         # otherwise use one of the available quality types.
         quality_type = sorted(list(available_quality_types))[0]
+        if self._global_container_stack is None:
+            Logger.log("e", "Global stack not present!")
```

Comment:
What do you think about creating a Logger.error() and/or Logger.e()?

Reply:
It could make it easier to discover which options are available, but it's not that complicated anyway.

I do think that it's not worth our time to change all of the logging calls in our code base now.

Reply:
It could make it easier to discover which options are available, but it's not that complicated anyway.

I do think that it's not worth our time to change all of the logging calls in our code base now.